README.md

```diff
@@ -43,10 +43,10 @@ For example, to run Keywhiz with an H2 database in development mode:
 
     # Run server
     java -jar $SERVER_JAR server $KEYWHIZ_CONFIG
+    
+The default username and password in development mode is `keywhizAdmin:adminPass`.    
```

Comment:
This is only true if you've run db-seed, which I've removed from the instructions -- you're now instructed to add your own admin user.